src/plugins/intel_cpu/src/nodes/reorder.cpp

```diff
@@ -189,11 +189,62 @@ void Reorder::createReorderPrimitive(const dnnl::memory::desc& srcDesc,
     if (!selectedPD)
         IE_THROW() << "Preferable primitive descriptor is not set.";
 
+    // How to query what data_types are supported by dnnl::reorder
```

Comment:
From the performance perspective it's better to use some a kind of a list of supported precisions. The serious drawback of this solution is the necessity to manually maintain such a list. May be this is not a big problem. Another thing is that oneDNN supports not all the precision combinations in the reorder implementation. Thus, we would have to maintain a list of supported combinations as well. Thus, may be we need to do the following. Maintain a list of supported by oneDNN openvino types (may be simply modify DnnlExtensionUtils::IEPrecisionToDataType so that it return a kind of error code instead of exception, so we can check whether the type is supported by oneDNN), then, when we proved that the current precisions are supported, we need to try to make the reorder and check the result, and if it is not created it means that the current precisions combination is not supported and we have to do the reordering with only one type and then perform the conversion.